build.go

```diff
@@ -335,33 +333,6 @@ func parseFlags() {
 	flag.Parse()
 }
 
-func setup() {
-	packages := []string{
-		"github.com/alecthomas/gometalinter",
```

Comment:
Maybe we could trim this down to the stuff actually still in use and existing? Is something we use, and many of the things below. I think metalinter itself has a command to install the rest of the tools it uses, now. Off the top of my head:

```
unused:
"github.com/AlekSi/gocov-xml",
"github.com/axw/gocov/gocov",
"github.com/FiloSottile/gvt",
"github.com/mitchellh/go-wordwrap",		

metalinter:
"golang.org/x/lint/golint", 
"github.com/gordonklaus/ineffassign",		
"github.com/mdempsky/unconvert",	
"github.com/opennota/check/cmd/...",		
"github.com/tsenart/deadcode",		
"golang.org/x/tools/cmd/cover",		
"honnef.co/go/tools/cmd/gosimple",		
"honnef.co/go/tools/cmd/staticcheck",		
"honnef.co/go/tools/cmd/unused",		

??? used by the translation stuff, but unclear if go-getting it here helps any more
"golang.org/x/net/html",

used by build:
"github.com/josephspurrier/goversioninfo",
```